ADALiOS/ADALiOS/ADURLProtocol.m

```diff
@@ -137,6 +137,7 @@ - (void)startLoading
     AD_LOG_VERBOSE_F(@"-[ADURLProtocol startLoading]", @"host: %@", [self.request.URL host]);
     
     NSMutableURLRequest* request = [self.request mutableCopy];
+    [request setTimeoutInterval:60.0];//set default timeout interval after copy
```

Comment:
According to my test, the above line makes no difference to the behaviour. 
In ADURLProtocol, no timeout will be triggered with or without the above line, or even set `[request setTimeoutInterval:10.0];`.

So I tried to keep the change "as minimal as possible", making sure I won't change any behavior of ADURLProtocol.